tests/browser/pe/program_enrollments.robot

```diff
@@ -0,0 +1,97 @@
+
+    #Go into Edit Mode - Note:  we are in Affiliations/Settings
+    Click Button                            Edit
+
+    #Wait Until Element Is visible           //label[@class='slds-checkbox']//input[@class='copy-start-date uiInput uiInputCheckbox uiInput--default uiInput--checkbox']//following-sibling::span[@class='slds-checkbox--faux']
+    Wait Until Element Is visible           (//label[@class='slds-checkbox'])//input[@class='copy-start-date uiInput uiInputCheckbox uiInput--default uiInput--checkbox']
+    Select checkbox                         (//label[@class='slds-checkbox'])//input[@class='copy-start-date uiInput uiInputCheckbox uiInput--default uiInput--checkbox']
+    Checkbox Should Be Selected             (//label[@class='slds-checkbox'])//input[@class='copy-start-date uiInput uiInputCheckbox uiInput--default uiInput--checkbox']
+
+    #Save settings
+    Click Button                            Save
+    
+
+
+
+    Wait Until Element Is visible           //a[contains(text(), 'Affiliation Mappings')]
+    Click Element                           //a[contains(text(), 'Affiliation Mappings')]
+
+    #Go into Edit Mode
+    Click Button                            Edit
+
+    #Save settings
+    Click Button                            Save
+
+
+Create A Contact
+
+
+    Go To Object Home                       Contact
+
+    Wait Until Element Is visible           //a[@title='New']//div[@title='New']
+    Click Element                           //a[@title='New']//div[@title='New']
+
+    Wait Until Element Is visible           //input[contains(@class,'firstName')]
+    Click Element                           //input[contains(@class,'firstName')]
+
+    Input Text                              //input[contains(@class,'firstName')]    firstName
+
+    Click Element                           //input[contains(@class,'lastName')]
+    Input Text                              //input[contains(@class,'lastName')]     lastName
+
+    Click Element                           //button[@title='Save']
+
+    Click Element                           (//following::div[@class='slds-no-flex']//div[@class='actionsContainer']//div[@title='New'])[4]
+
+    Wait Until Element Is visible           //div[@class='autocompleteWrapper slds-grow']//input[@class=' default input uiInput uiInputTextForAutocomplete uiInput--default uiInput--input uiInput uiAutocomplete uiInput--default uiInput--lookup']
+    Populate Field                          Program    lastName Administrative Account
+
+    #Create a New Account as part of this flow
+    Click Element                           //span[@title='New Account']
+
+    #Drop down and select Academic
+    Wait Until Element Is visible           //button[@class='slds-button slds-button--neutral uiButton--default uiButton--brand uiButton']//span[@class=' label bBody']
+    Sleep                                   5
```

Comment:
Can we get rid of this sleep since you are anyway using "Wait until element is visible" ?